pkg/scheduler/apis/config/v1alpha1/conversion_test.go

```diff
@@ -191,3 +193,206 @@ func TestConfigToV1alpha1KubeSchedulerLeaderElectionConfiguration(t *testing.T)
 		}
 	}
 }
+
+func TestConvertBetweenV1Alpha1PluginsAndConfigPlugins(t *testing.T) {
+	// weight is assigned to score plugins
+	weight := int32(10)
+	// DummyWeight is a palceholder for the v1alpha1.plugins' weight will be filled with zero when
+	// convert back from config.
+	dummyWeight := int32(0)
```

Comment:
`Weight` can be nil. Just put it for Score plugins, where it is actually used.

Reply:
When convert from framework config, it will be filled with 0, not nil.
Weight in config is int, not *int.

Reply:
Uhm... true. Well, it's not really important during conversion. Feel free to leave it. But maybe put a value so that we know it's not using the default (0)

Reply:
Ok, a non-zero value makes sense.